apps/src/storage/dataBrowser/LibraryCategory.jsx

```diff
@@ -43,31 +45,31 @@ class DataCategory extends React.Component {
     const icon = this.state.collapsed ? 'caret-right' : 'caret-down';
     return (
       <div>
-        <div onClick={this.toggleCollapsed}>
+        <a style={styles.categoryName} onClick={this.toggleCollapsed}>
           <FontAwesome icon={icon} />
-          <span style={styles.categoryName}>{this.props.name}</span>
+          <span>{this.props.name}</span>
           <span style={styles.tableNumber}>
             {this.props.datasets.length}{' '}
             {this.props.datasets.length === 1 ? 'table' : 'tables'}
           </span>
-        </div>
+        </a>
```

Comment:
It felt a little weird that it acted like a link until I clicked on it. I was expecting to be taken to a new page rather than have the drop-down expand. I wonder if just having the caret is enough to take care of the accessibility issue. This might be a good question for Erin. 

That being said, will this likely change when you get the design from Mark?

Reply:
After chatting offline, the accessibility that `<a>` improves is for screenreaders - they notice buttons and `<a>` tags, but not clickable `<div>` tags. I take back my comment :) Also, the styling from Mark will improve the "feel" of the element so it doesn't seem like it will open a new webpage.